executor/analyze.go

```diff
@@ -962,10 +974,14 @@ func (e *AnalyzeFastExec) handleSampTasks(bo *tikv.Backoffer, workID int, err *e
 		for _, key := range keys {
 			var iter kv.Iterator
 			iter, *err = snapshot.Iter(key, endKey)
+			fastAnalyzeCounterSeekKeys.Inc()
 			if *err != nil {
 				return
 			}
 			if iter.Valid() {
+				if _, ok := kvMap[string(iter.Key())]; !ok {
+					fastAnalyzeCounterSample.Inc()
```

Comment:
I guess it will increase forever as long as there are fast analyze queries, and it will provide little info for us. What about using a histogram and only update it at the end of a query?